js/weechat.js

```diff
@@ -647,6 +648,10 @@
         if (params.password !== null) {
             keys.push('password=' + params.password);
         }
+        if (!!params.totp) {
+            keys.push('totp=' + params.totp);
+        }
+        console.log(keys);
```

Comment:
Presumably debugging that should go away before the branch is ready?

Reply:
certainly, this is more of a proof of concept than a pull request at this stage :)